src/components/image_preview_modal.js

```diff
@@ -26,7 +25,7 @@ export default class ImagePreviewModal extends React.Component {
       <Modal size="lg" show={this.state.show} onHide={this.handleClose}>
         <Modal.Header closeButton>
           <Modal.Title as="h5">
-            Image Preview - {Path.basename(this.props.url)}
+            Image Preview - {this.props.url.split('/').pop()}
```

Comment:
I think `Path.basename()` conveys the concept more clearly but I am assuming it's not available for some reason?

Reply:
Correct - older versions of webpack included polyfills for backend modules like Path, so it likely compiled to something like this. Upgrading to Node 20 excludes that polyfill unelss I explicitly include it, but I am also trying to lighten the dependency tree and this seemed like a trivial need.